django_netjsonconfig/base/base.py

```diff
@@ -135,15 +135,21 @@ def get_backend_instance(self, template_instances=None):
         """
         backend = self.backend_class
         kwargs = {'config': self.get_config()}
+        default_values = {}
```

Comment:
it's a bit confusing, this is a generic method, we need to keep the naming generic. Default values is something we are adding for templates.

I would rename this to `context`